src/engine/source/proto/src/catalog.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto3";
+
+import "engine.proto";
+
+package com.wazuh.api.engine.catalog;
+
+/*
+ * This API is used to manage the catalog of the Wazuh manager
+ */
+
+/* Supported formats for the resources (Case sensitive) */
+enum ResourceFormat
+{
+    option allow_alias = true;
+    json = 0; // JSON format
+    yaml = 1; // YAML format
+    yml = 1;  // Alias for YAML
+}
+
+/*
+ * Supported types for the resources (Case sensitive)
+ * UNKNOWN is only used for errors
+ */
+enum ResourceType
+{
+    UNKNOWN = 0;     // Unknown (Only for errors)
+    decoder = 1;     // Decoder
+    rule = 2;        // Rule
+    filter = 3;      // Filter
+    output = 4;      // Output
+    environment = 5; // Environment
+    schema = 6;      // Schema
+    collection = 7;  // Collection
+}
+
+/***************************************************
+ * Post a resource in the catalog
+ *
+ * command: catalog.resource/post (<resource>/<accion>)
```

Comment:
replace `accion` for _action_, same in the documentation

Reply:
Fixed